sh.loft.Devpod.yaml

```diff
@@ -0,0 +1,50 @@
+id: sh.loft.Devpod
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: DevPod
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --device=dri # OpenGL, not necessary for all projects
+  - --share=ipc
+  - --share=network
+  - --socket=ssh-auth
+  - --socket=gpg-agent
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Flatpak
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/loft-sh/devpod/releases/download/v0.6.10/DevPod_0.6.10_amd64.deb
+        sha256: d43738efa60735cc4ad6b63285f9e449534831ba9a2c7ac2c72cb682fb26f4b5 # This is required if you are using a remote source
+        only-arches: [x86_64] #This source is only used on x86_64 Computers
+        # This path points to the binary file which was created in the .deb bundle.
+        # Tauri also creates a folder which corresponds to the content of the unpacked .deb.
+      - type: file
+        path: devpod-wrapper
+      - type: file
+        path: sh.loft.Devpod.metainfo.xml
+      - type: file
+        path: sh.loft.Devpod.desktop
+    build-commands:
+      - ar -x *.deb
+      - tar -xf data.tar.gz
+      - cp devpod-wrapper /app/bin/devpod-cli
+      - chmod +x /app/bin/devpod-cli
+      - install -Dm755 usr/bin/devpod-cli /app/bin/devpod-bin
+      - install -Dm755 usr/bin/DevPod /app/bin/DevPod
+      - install -Dm644 sh.loft.Devpod.desktop /app/share/applications/sh.loft.Devpod.desktop
```

Comment:
so the .deb doesn't have a .desktop? that's a bug.

Reply:
moving this to upstream to use the generate .desktop